src/IB/IIMethod.cpp

```diff
@@ -2932,8 +3052,8 @@ IIMethod::spreadForce(const int f_data_idx,
 
     for (unsigned int part = 0; part < d_num_parts; ++part)
     {
-        PetscVector<double>* X_vec = d_X_half_vecs[part];
-        PetscVector<double>* X_ghost_vec = d_X_IB_ghost_vecs[part];
+        PetscVector<double>* X_vec = getMeshCoordinatesPetsc(d_use_current_mesh_configuration, "half", part); //d_X_half_vecs[part];
```

Comment:
```suggestion
        auto X_vec = static_cast<PetscVector<double*>(getMeshCoordinates(d_use_current_mesh_configuration, "half", part));
```

Reply:
I see, this allows for removal of the Petsc function 👍